concordium-std-derive/src/lib.rs

```diff
@@ -998,6 +1006,7 @@ fn receive_worker(attr: TokenStream, item: TokenStream) -> syn::Result<TokenStre
         parameter_option,
         return_value_option,
         error_option,
+        None, // Event schema should be attached to the `init` function and is globally available.
```

Comment:
This will likely change, assuming you change the COntractV3 modelling in contracts-common.